libs/community/langchain_community/vectorstores/__init__.py

```diff
@@ -334,6 +337,7 @@
     "Clickhouse",
     "ClickhouseSettings",
     "CouchbaseVectorStore",
+    "CrateDBVectorSearch",
```

Comment:
Last call about »naming things« matters: `CrateDBVectorSearch` vs. `CrateDBVectorStore`?

Current cardinalities, amongst implementations from other vendors:
- VectorSearch: 9
- VectorStore: 9

I am tending to rename the class to `CrateDBVectorStore`, what do others think about it?

Reply:
The classes have been renamed to `CrateDBVectorStore` resp. `CrateDBVectorStoreMultiCollection`.